### DISCUSSION TITLE
Problem!

### DISCUSSION THREAD

**Discussion by Twdmaster** [skyline-emu/skyline#1824] [Q&A]
How to play 2nd, 3rd and 4th episodes of the walking dead final season?
I downloaded nsp for them, but they don't work( 
Does nsp/xci with all episodes exist?
Plz help!!!
I am playing twd using egg ns emulator, I hope this game will accessable on skyline!

**Comment by dima-xd** [comment]
Downloaded, understandable

**Comment by guranu** [comment]
You realize that we don't like it when pirate a game.
